services/app-api/forms/2026/measureTemplates.ts

```diff
@@ -1293,6 +1406,62 @@ export const measureTemplates: Record<
       measureFooter,
     ],
   },
+  [MeasureTemplateName["FFS-POM-3"]]: {
+    id: "FFS-POM-3",
+    title: "POM: People choose services (FFS LTSS)",
+    type: PageType.MeasureResults,
+    sidebar: false,
+    elements: [
+      {
+        type: ElementType.ButtonLink,
+        id: "return-button",
+        label: "Return to Measure Information",
+        to: "POM-3",
+      },
+      {
+        ...measureHeader,
+        text: "POM: People choose services (FFS LTSS)",
+      },
+      measureInstructions,
+      feeForServiceMeasureResultsSubheader,
+      ...whichMedicaidHCBSprograms,
+      performanceRatePOM,
+      {
+        type: ElementType.MeasureFooter,
+        id: "measure-footer",
+        prevTo: "POM-3",
+        completeSection: true,
+      },
+    ],
+  },
+  [MeasureTemplateName["MLTSS-POM-3"]]: {
+    id: "MLTSS-POM-3",
+    title: "POM: People choose services (MLTSS)",
+    type: PageType.MeasureResults,
+    sidebar: false,
+    elements: [
+      {
+        type: ElementType.ButtonLink,
+        id: "return-button",
+        label: "Return to Measure Information",
+        to: "POM-3",
+      },
+      {
+        ...measureHeader,
+        text: "POM: People choose services (MLTSS)",
+      },
+      measureInstructions,
+      feeForServiceMeasureResultsSubheader,
+      ...whichMedicaidHCBSprograms,
+      performanceRatePOM,
+      {
+        type: ElementType.MeasureFooter,
+        id: "measure-footer",
+        prevTo: "POM-3",
+        completeSection: true,
+      },
+    ],
+  },
   [MeasureTemplateName["POM-4"]]: {
     id: "POM-4",
     title: "POM: People Realize Personal Goalss",
```

Comment:
This was already there, but while we're fixing typos - might as well - lol
```suggestion
    title: "POM: People Realize Personal Goals",
```